fineract-provider/src/main/java/org/apache/fineract/accounting/glaccount/data/GLAccountData.java

```diff
@@ -132,14 +135,6 @@ private GLAccountData(String name, Long parentId, String glCode, Boolean manualE
         this.allowedExpensesTagOptions = null;
     }
 
-    public Integer getRowIndex() {
-        return rowIndex;
-    }
-
-    public CodeValueData getTagId() {
-        return tagId;
-    }
-
     public GLAccountData(final Long id, final String name, final Long parentId, final String glCode, final boolean disabled,
```

Comment:
Replace with "@RequiredArgsConstructor"?

Reply:
annotating with @RequiredArgsConstructor gives an error at lines 194-195. 

error : cannot resolve constructor

I guess the constructor is not generated here even after applying @RequiredArgsConstructor.

Reply:
@hmg65 you know that when you add the annotation for the constructor then you have to remove the existing? Other than that I don't think another error is possible... this is pretty straight forward...